build-support/pants_venv

```diff
@@ -39,10 +40,10 @@ function create_venv() {
   "${REPO_ROOT}/build-support/virtualenv" "$(venv_dir)"
 }
 
-function ensure_gcc() {
-  if ! GCC_VERSION="$(gcc -v 2>&1)"; then
+function ensure_c_compiler() {
+  if ! GCC_VERSION="$(${CC} -v 2>&1)"; then
```

Comment:
Nit, but how about `CC_VERSION`? If you read the use below it probably should have been so-named from the get-go and perhaps that would've triggered thought about whether to rely on `gcc` being an alias to `clang` which it apparently was for the author.